odc_eks/variables.tf

```diff
@@ -167,13 +185,60 @@ variable "enable_nat_gateway" {
   default     = true
 }
 
+variable "single_nat_gateway" {
+  description = "Should be true if you want to provision a single shared NAT Gateway across all of your private networks"
+  type        = bool
+  default     = false
+}
 
 variable "create_igw" {
   type        = bool
   description = "Whether to provision an Internet Gateway in the VPC. Default is true (False for private routing)"
   default     = true
 }
 
+variable "create_vpc_flow_logs" {
+  type        = bool
+  description = "Whether to create VPC flow logs. Default is set to 'false'"
+  default     = false
+}
+
+variable "flow_log_max_aggregation_interval" {
```

Comment:
I feel like these should be consistent so if we've got `create_vpc_flow_logs` we should use `vpc_flow_log_max_aggregation_interval`, etc.

Reply:
If the variable is just being passed through to the AWS module, I've tried to keep the same attribute name.  Point taken though, so I've renamed to `create_vpc_flow_logs` to  `create_flow_log`